src/screens/DashboardTx.js

```diff
@@ -92,6 +141,8 @@ class DashboardTx extends React.Component {
 
     return (
       <div className="content-wrapper">
+        <p className="mt-4 mb-4">Here you can see the most recent transactions and blocks of the network.</p>
+        <SearchTx {...this.props} newData={this.newData} resetData={this.resetData} />
```

Comment:
I think we should use the search text field at the top of the explorer. Then, after searching, we can show it (or not). What do you think?

Reply:
At first I thought it would be weird to have this search on top in the mobile but I will give it a try, probably is better for usability

Reply:
If we can have both on the address bar it's better for saving space on the main screen. But I'm also concerned about how it'll look on mobile.